### ISSUE TITLE
403 Error

### ISSUE THREAD

**Issue by BambouJumpOnYou** [Sprayxe/valorant.js#20]
Hello,

I have a problem with the package.
When I put my credentials just to do a simple request I get a 403 response : 
![image](https://user-images.githubusercontent.com/46085740/154057122-9f933a8b-2830-4732-bbf0-53178837e186.png)

I also tried to create another account to see if I got the same problem and the answer is yes

Am I doing something wrong ?

**Comment by BobbyESP** [comment]
I dont think so. The library is deprecated and Riot recently changed the ayth system, so a refactor is needed.

**Comment by BobbyESP** [comment]
Maybe i try to fix it
